com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/ZipUtil.java

```diff
@@ -46,55 +46,45 @@ private static String stripBaseDir(File baseDir, File file) {
 		return strippedPath;
 	}
 
+	private static CRC32 calculateCrc32(File file) throws IOException {
```

Comment:
Moved crc calculation into a separate step. We need to do this since we can't necessarily load the entire file in one go before writing it into the ZipEntry (and we need the crc32 before writing the zip entry)